src/templates/community-champion-profile.js

```diff
@@ -15,9 +42,37 @@ const CHAMPION_PROFILE_BREADCRUMBS_PREFIX = [
 ];
 
 const ContentContainer = styled('div')`
-    padding: ${size.default} ${size.xxlarge};
-    @media ${screenSize.upToLarge} {
-        padding: ${size.mediumLarge} ${size.default};
+    padding: ${size.default} ${size.xxlarge} ${size.xlarge};
+    @media ${screenSize.upToSmallDesktop} {
+        padding: ${size.mediumLarge} ${size.default} ${size.xlarge};
+    }
+`;
+
+const ChampionImage = styled(ProfileImage)`
+    div {
+        ${props =>
+            props.default
+                ? ''
+                : `background-color: ${props.theme.colorMap.devWhite};`}
```

Comment:
added this here because the white background colour was showing behind the default profile pic
![Screen Shot 2021-08-03 at 3 16 26 PM](https://user-images.githubusercontent.com/46455728/128082090-f3d56013-d893-4a9e-b587-ea6d2dd8241f.png)

Reply:
ah so is the goal to have some background color?

Reply:
yes, this is because one of the photos has a transparent background - https://github.com/mongodb/devhub/pull/902